source/application/Utils/File.cpp

```diff
@@ -186,12 +186,12 @@ Utils::File::ExecutableFileHeaders::ExecutableFileHeaders(const std::wstring& fi
             //}
 
 
-DWORD Utils::File::ExecutableFileHeaders::GetImageSizeInRAM() const
+DWORD Utils::File::ExecutableFileHeaders::GetImageSizeInRAM()

```

Comment:
This is because the code now accesses the buffer directly. There's a lot of stupid `const_cast` necessary otherwise, because I can't cast the pointer. If you have a nicer way to do it, please show me.